enterprise/api/v1/views/enterprise_course_enrollment.py

```diff
@@ -86,3 +96,89 @@ def get_serializer_class(self):
         if self.request.method in ('GET',):
             return serializers.EnterpriseCourseEnrollmentWithAdditionalFieldsReadOnlySerializer
         return serializers.EnterpriseCourseEnrollmentWriteSerializer
+
+
+class EnterpriseCourseEnrollmentAdminPagination(PageNumberPagination):
+    """
+    Custom pagination class for Enterprise Course Enrollment API.
+    """
+    page_size = 10
+    page_size_query_param = 'page_size'
+    max_page_size = 100
+
+
+class EnterpriseCourseEnrollmentAdminViewSet(EnterpriseReadWriteModelViewSet):
+    """
+    API views for the ``enterprise-course-enrollment-admin`` API endpoint.
+    """
+
+    queryset = models.EnterpriseCourseEnrollment.with_additional_fields.all()
+    serializer_class = serializers.EnterpriseCourseEnrollmentAdminViewSerializer
+    permission_classes = (permissions.IsAuthenticated, permissions.IsAdminUser)
+    pagination_class = EnterpriseCourseEnrollmentAdminPagination
+
+    @action(detail=False, methods=['get'])
+    def get_enterprise_course_enrollment(self, request):
+        """
+        Endpoint to get enrollments for a learner by `lms_user_id` and `enterprise_uuid` viewed
+        by an admin of that enterprise.
+
+        Parameters:
+        - `lms_user_id` (str): Filter results by the LMS user ID.
+        - `enterprise_uuid` (str): Filter results by the Enterprise UUID.
+        """
+        lms_user_id = request.query_params.get('lms_user_id')
+        enterprise_uuid = request.query_params.get('enterprise_uuid')
+        if not lms_user_id or not enterprise_uuid:
+            return Response(
+                {"error": "Both 'lms_user_id' and 'enterprise_uuid' are required parameters."},
+                status=HTTP_400_BAD_REQUEST
+            )
+        enterprise_customer_user = get_object_or_404(
+            models.EnterpriseCustomerUser,
+            user_id=lms_user_id,
+            enterprise_customer__uuid=enterprise_uuid
+        )
+        enterprise_enrollments = models.EnterpriseCourseEnrollment.objects.filter(
```

Comment:
Could you test whether the enrollments are filtered correctly? Perhaps can be extracted into a function to test more easily